Sync Discord Bot Template

tenzyu opened the pull request:
# about

- Remove unused module(flake8:F401)
- Resolve do not use bare 'except'(flake8:E722)

tenzyu committed:
initialize

tenzyu committed:
put vscode settings

tenzyu committed:
minor fix

tenzyu committed:
wrote language and extension

tenzyu committed:
renamed mod -> cogs_controller

tenzyu committed:
added pylintrc

AttoCat committed:
Refactoring

tenzyu committed:
Merge pull request #4 from AttoCat/master

Refactoring

tenzyu committed:
Merge remote-tracking branch 'template/master' into sync-bot-template

Kesigomon reviewed (approved):
![0001](https://user-images.githubusercontent.com/34025522/105635644-4da2db00-5ea7-11eb-97f7-fd5c80b790bd.png)
LGTM!

tenzyu committed:
minor fix

tenzyu committed:
minimal importing and shorten constant

tenzyu committed:
minor fix

tenzyu committed:
updated logged in message

tenzyu committed:
renamed function names

tenzyu committed:
Merge pull request #5 from tenzyu/refactoring

Refactoring

tenzyu committed:
Merge remote-tracking branch 'template/master' into sync-bot-template

tenzyu commented:
# about 2
- minor fixes (ex. added end line)
- rename function names in cogs_controller.py
- minimal importing
- shorten constant names
- updated logged in message

だいたいコミットメッセージの通りです

AttoCat reviewed (approved):
LGTM
